migrations/sql/V2020.04.01.10.33__add_now_statuses.sql

```diff
@@ -0,0 +1,23 @@
+-- Create a new column for defining the desired display order of the statuses
+ALTER TABLE now_application_status ADD COLUMN display_order integer;
+
+-- Delete all of the old Core statuses (Accepted, Under Review, and Withdrawn)
+DELETE FROM now_application_status WHERE now_application_status_code = 'ACC';
```

Comment:
I think this will fail when deployed to test. In test, these statuses do exist and are set. So I think the appropriate order of operations here is:

1. Insert new statuses
2. Set any applications to the default new status (Active/Issued/Approved)
3. Delete the old statuses

Reply:
Good point, fixed!